morf-core/src/main/java/org/alfasoftware/morf/jdbc/DatabaseMetaDataProvider.java

```diff
@@ -1048,9 +1048,7 @@ protected static AName named(String name) {
    * @param schemaName
    * @return
    */
-  protected String buildSequenceSql(String schemaName) {
-    return null;
-  }

```

Comment:
Note: I like this change, but it also makes the class abstract, which is a backwards-incompatible change.
That said, there were other changes already here which were backwards-incompatible too, although in less visible places.